app-backend/datamodel/src/main/scala/User.scala

```diff
@@ -293,7 +293,8 @@ case class UserBulkCreate(
     campaignId: Option[UUID] = None,
     grantAccessToParentCampaignOwner: Boolean = false,
     grantAccessToChildrenCampaignOwner: Boolean = false,
-    copyResourceLink: Boolean = false
+    copyResourceLink: Boolean = false,
+    isAltConnection: Option[Boolean] = None
```

Comment:
This is set to an `Option` and given a `None` so that we don't need to update data model on frontend app that does not need this change.

Reply:
Looks indeed smelly, don't you want to introduce an ADT to handle it? It can be really confusing later to remember the semantics of `Option[Boolean]`

Smth like: 

```scala
sealed trait IsAltConnection
case object AltConnection extends IsAltConnection
case object NotAltConnection extends IsAltConnection
case object DefaultConnection extends IsAltConnection

// + extra codecs to convert into boolean
```

Reply:
https://github.com/raster-foundry/raster-foundry/search?q=Option%5BBoolean%5D we're cursed, disregard this comment. ):